solutions/crystal/01-init/code/.gitignore

```diff
@@ -0,0 +1,2 @@
+lib/
+server
```

Comment:
Binaries typically get placed in `bin/` or so for most languages - what's the default that Crystal uses?

Reply:
lib/ is where the dependencies are stored.
And server is the final artifact. (We are passing this name with the -o flag).

Reply:
No other build artifacts would be there in the /app dir.